lib/client/getProfile.js

```diff
@@ -2,10 +2,14 @@ const _validate = require('../_validate')
 
 const CONTEXT = 'getCustomerProfile'
 
-module.exports = async (xhr, contextId) => {
+module.exports = async (xhr, contextId, params = {}) => {
   _validate({ xhr, contextId }, CONTEXT)
 
-  const { data } = await xhr.get('/user/profile', {
+  const qs = Object.keys(params)
+    .map(field => `${encodeURI(field)}=${encodeURI(params[field])}`)
+    .join('&')
```

Comment:
logica asta nu o putem pune in xhr.get ?

Reply:
banuiesc ca o sa se mai repete

Reply:
`xhr` este cam orb, nu se ocupa de procesare decat in ceea ce priveste header-ele.